jobsapp/views/home.py

```diff
@@ -71,3 +76,21 @@ def get(self, request, *args, **kwargs):
             raise Http404(_("Job doesn't exists"))
         context = self.get_context_data(object=self.object)
         return self.render_to_response(context)
+
+
+class ContactView(FormView):
+    template_name = "contact_us.html"
+    form_class = ContactForm
+    success_url = "/"
+
+    def post(self, request: HttpRequest, *args: str, **kwargs: Any) -> Union[HttpResponse, Any]:
```

Comment:
Override post method is not necessary.
Put all logical inside `form_valid` and `form_invalid` methods.